CHANGELOG.md

```diff
@@ -20,6 +20,9 @@ a release.
 
 ## [Unreleased]
 
+### Fixed
+- Add compatibility with doctrine/orm > 2.10 (#2272)
```

Comment:
```suggestion
- Add compatibility with doctrine/orm >= 2.10 (#2272)
```

Reply:
Friendly ping @franmomu.

Reply:
I thought I did it 😕 I'll be AFK today, if you want to change it or I can do it tomorrow